data/ubaniIsaac.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Isaac Ubani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Just some guy that likes to code and do calisthenics",
+   "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/i_ce_u" },
+    { "icon": "FaGithub", "url": "https://github.com/ubaniIsaac" },
+    { "icon": "FaLink", "url": "https://iceu.netlify.app" }
+  ],
+  "avatar": "https://github.com/ubaniIsaac.png",
+  "links": [
+     {
+      "name": "Here's a link to my portfolio",
+      "url": "iceu.netlify.app",
```

Comment:
It's better to define the `https` extension for the link, because it sometimes use the domain of the app host. Something like this on click.
![image](https://user-images.githubusercontent.com/88786642/211163795-793ca865-6735-4193-889f-11a0a597962b.png)


```suggestion
      "url": "https://iceu.netlify.app",
```